src/django/api/serializers.py

```diff
@@ -430,13 +431,33 @@ def get_oar_id(self, facility):
     def get_country_name(self, facility):
         return COUNTRY_NAMES.get(facility.country_code, '')
 
+    def get_contributors(self, facility):
+        def format_source(source):
+            if type(source) is Source:
+                return {
+                    'id': source.contributor.admin.id
+                    if source.contributor else None,
+                    'name': source.display_name,
+                    'is_verified': source.contributor.is_verified
+                    if source.contributor else False,
+                }
+            return {
+                'name': source,
+            }
+        request = self.context.get('request') \
+            if self.context is not None else None
```

Comment:
We use this construction a few places in models.py too. In general I tend to prefer to use explicit `None` checks when I can to avoid type coercion ambiguity, like empty objects being falsey.

```python
>>> if {'key': 1}:
...   print('true')
...
true
>>> if {}:
...   print('true')
...
>>>
```